docs/docs/docs/01-core/content-releases/02-frontend/02-details-view.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Release details view
+tags:
+  - content-releases
+  - tech design
+---
+
+### Overview:
+
+The details-view offers a comprehensive presentation of all entries linked to the release. These entries are meticulously organized into tables based on various options. Each entry's status is clearly indicated, providing the ability to edit or remove entries from the release.
+
+On this page, users with the appropriate permissions can perform key actions such as publishing the release, editing its details, or deleting it.
+
+#### Grouping entries:
+
+Entries associated with a release can be systematically organized in tables using various grouping options:
+
+- <strong>by Content-Types</strong>: Organize entries by collections.
+- <strong>by Locales</strong>: Group entries based on locale.
+- <strong>by Actions</strong>: Categorize entries by action to perform, making it easier to discern
+  between entries to publish and entries to unpublish.
+
+#### Edit a release:
+
+Users have the flexibility to change the name of a release, by simply clicking on the "More" button, the Edit button is revealed (enabled if the user role has `plugin::content-releases.update` permission).
+The Edit button opens a modal, enabling users to modify the release name. To save the changes, the new name must be unique, non-empty, and distinctly modified.
+
+#### Delete a release:
+
+Deleting a release is a straightforward process for users. By clicking on the "More" button, the "Delete" button is exposed (enabled if the user role has `plugin::content-releases.delete` permission).
+
+Upon activation, the "Delete" button triggers a confirmation modal, prompting the user to confirm the deletion.
```

Comment:
Same as above and I think the action is very similar to the edit one so maybe the description here could be as similar.